packages/react/test/specs/lib/felaRenderKeyframesPlugin-test.ts

```diff
@@ -0,0 +1,59 @@
+import felaRenderKeyframesPlugin from 'src/lib/felaRenderKeyframesPlugin'
+import { felaRenderer } from 'src/lib'
+
+const renderKeyframesPlugin = felaRenderKeyframesPlugin()
+
+describe('felaRenderKeyframesPlugin', () => {
+  test('does not transform the animationName prop if it is already string', () => {
+    const style = {
+      animationName: 'k1',
+      animationDuration: '2s',
+    }
+
+    expect(renderKeyframesPlugin(style, 'RULE', felaRenderer)).toMatchObject(style)
+  })
+
+  test('transforms the animationName prop if it contains keyframe in the definition', () => {
+    const style = {
+      animationName: {
+        keyframe: () => ({ from: { rotate: '0deg' }, to: { rotate: '360deg' } }),
```

Comment:
I am confused with this stuff. Do we really want to implement something that is different from Fela APIs?

https://github.com/rofrischmann/fela/blob/master/packages/fela-plugin-embedded/src/index.js#L45

Reply:
I'm open to suggestions, we need to still use the keyframe params object, we don't want to restrict the API with this changes.

Reply:
Let's keep existing for this PR, but we definitely should move to `fela-plugin-embedded` and align APIs.

Reply:
We can adjust the plugin to convert the Stardust's animation API to the one required by the fela embedded plugin and preserve our functionality as well as use the embedded plugin.